cmd/kpod/docker/types.go

```diff
@@ -0,0 +1,271 @@
+package docker
```

Comment:
I had originally cut and paste because that's what buildah did, I can use the original but I thought there might be some reason why this wasn't done in the first place